packages/entity-server/src/routes/hierarchy/MultiEntityRoute.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Route } from '@tupaia/server-boilerplate';
+import { formatEntitiesForResponse } from './format';
+import { MultiEntityRequest } from './types';
+
+export class MultiEntityRoute extends Route<MultiEntityRequest> {
+  async buildResponse() {
+    const { entities, field, fields } = this.req.ctx;
+    return formatEntitiesForResponse(this.req.models, this.req.ctx, entities, field || fields);
+  }
+}
```

Comment:
Surprised we didn't already have this route! It's needed by the report-server to check if the user has access to the entities they've requested data for...